cs/src/core/Bond.mono.csproj

```diff
@@ -0,0 +1,121 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectGuid>{43CBBA9B-C4BC-4E64-8733-7B72562D2E91}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Bond</RootNamespace>
+    <AssemblyName>Bond</AssemblyName>
+    <BondRedistributable>true</BondRedistributable>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
If Mono is equivalent of .Net 4.5 we should add:

    <DefineConstants>$(DefineConstants);NET45</DefineConstants>

There is some conditional code (I think at the moment it is only aggressive inlining attribute on methods).